x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/grok.test.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import {
+  uiSettingsServiceMock,
+  i18nServiceMock,
+} from '../../../../../../../../../../src/core/public/mocks';
+
+import { Form, useForm, KibanaContextProvider } from '../../../../../../shared_imports';
+import { Grok } from './grok';
+
+// @ts-ignore
+window.Worker = function () {
+  this.postMessage = () => {};
+  (this as any).terminate = () => {};
+};
+
+// disable all react-beautiful-dnd development warnings
+(window as any)['__react-beautiful-dnd-disable-dev-warnings'] = true;
```

Comment:
nit: just so we don't affect other test it'd be better to have this inside a 

```
beforeAll
(() => {
  (window as any)['__react-beautiful-dnd-disable-dev-warnings'] = true;
});
```

and then have this

```js
afterAll(() => {
  (window as any)['__react-beautiful-dnd-disable-dev-warnings'] = false;
}):
```

just to clean up after ourselves 😊